client/templates/globals/connectioninfo.js

```diff
@@ -16,5 +24,26 @@ Template.connectionInfo.events({
         evt.preventDefault();
         console.log("Trying to reconnect...");
         Meteor.reconnect();
+    },
+
+    "click #btnWarningExpandCollapse": function (evt) {
```

Comment:
I think you can simplify this whole block. The following lines should do the very same:

```
"click #btnWarningExpandCollapse": function (evt) {
    evt.preventDefault();
    let warningMessage = document.getElementById("warningMessage");
    warningMessage.style.display = (warningMessage.style.display === "none") ? "inline-block" : "none";
}
```

This will also resolve the currently occurring client-side exception which will be shown whenever the info icon will be clicked.